src/collections/blog/2023/2023-05-24-meet-the-meshmate-yash-sharma/index.mdx

```diff
@@ -0,0 +1,192 @@
+---
+title: "Meet the MeshMate: Yash Sharma"
+subtitle: "An interview series with Layer5 MeshMates"
+date: 2023-05-24 10:30:05 -0530
```

Comment:
```suggestion
date: 2023-05-29 10:30:05 -0530
```

Reply:
@vishalvivekm Let's update the date here, and merge.

Reply:
Done @chadha19